src/views/ReturnDetails.vue

```diff
@@ -35,24 +43,33 @@
                 <p>{{ getProduct(item.productId).productId }}</p>
               </ion-label>
             </ion-item>
-            <ion-item class="product-count">
+            <ion-item :disabled="true" lines="none">
```

Comment:
do we need an item here? why can't we just put a chip here

Reply:
We can do that sir. Fixed it.